docs/settings.rst

```diff
@@ -58,6 +59,11 @@ ENCRYPT_BACKUP_CODES
 Backup codes to be encrypted before saving.
 Default: ``True``
 
+SECRET_KEY_LENGTH
+"""""""""""""""""
+Length of the shared secret key.  Please note: Google Authenticator on iOS requires this to be 16.
```

Comment:
Could you share source of this info?

Reply:
I have the Google Authenticator app on iOS.  When you go into manual entry the sample key they have is 16 characters.  If you try and use a key that is less or more than that, it gives you an error saying that it's not a valid key.  I'll try and find some technical documentation that backs it up but the default of 20 does not work for iOS.

Reply:
Looks like this is wrong!  I was able to use an 8 character code manually and a 24 character code.  It appears that they need to be in multiples of 8.  I'll update the docs to reflect that.  Unless there is a different preference for how to communicate that information?

Reply:
Just for the record, have you tried putting 'a' as a last character over 16? Some sources say that it works, what could potentially suggest there is a bug in GA app for iOS.

Reply:
That does work.  I would agree, that does appear to be a bug in the GA app for iOS.  But the multiples of 8 works even when the last character isn't 'a'.  Should I update the docs to indicate that requirement?

Reply:
Hi, here is an explanation of this issue: 
https://github.com/antonioribeiro/google2fa#google-authenticator-secret-key-compatibility

Reply:
Are you sure you are talking about secret key length and not code length?

Reply:
@mostaszewski very helpful link!  That explains what I'm seeing exactly.  @wgonczaronek yup, it's the secret key length.  When I change the hard coded length from 20 to 16 on my local copy, I'm able to add the secret key to my GA app.  If it's a code that is 20 characters long it doesn't work.

Based upon the link that's provided there, I'll update the documentation to specify that it needs to be at least 8 characters and a power of 2 (at least for iOS, it sounds like that's not a requirement on Android?) if that works for you all?

Reply:
I think direct reference to google's documentation would be more helpful.

Reply:
@lanthos I think we'd rather stick to 16 as a minimum or even a default (as it is in pyotp defualts). Android app doesn't accept anything less than 16.

Reply:
@tallpol3 ok cool.  Would it make sense to not have this as a variable thing then?  Since the minimum required on the Android app is 16 and that works for iOS as well?  Or do do you want to still have the variable but with the direct link to the Google documentation with a quick summary and the default being 16?

Reply:
I'm doing the latter and pushing that now but am happy to make any other changes as you see fit.